samples/peripheral/radio_test/src/radio_test.c

```diff
@@ -13,6 +13,15 @@
 #include <nrfx_timer.h>
 #include <zephyr.h>
 
+#if CONFIG_NRF21540_FEM
+#include "nrf21540.h"
+#endif
+
+/* Radio fast ramp up time in us. */
+#define FAST_RAMP_UP_TIME 40
+/* Radio normal ramp up time in us */
+#define RAMP_UP_TIME 130
```

Comment:
Consider moving these definitions to nrf21540.h instead of duplicating it in both DTM and radio test source code.

Reply:
Moved to nrf21540.c